docs/model_cache.md

```diff
@@ -42,6 +42,8 @@ In case there are valid reasons to enable the model cache also for models with a
 
 > IMPORTANT: Models imported via the custom loaders never create or use any cache.
 
+> IMPORTANT: Model cache can't be used with HDDL devices.
```

Comment:
Model cache can be used with HDDL, but doing so causes service crash, is that correct? There is also "Troubleshooting" document for such cases. @dtrawins

Reply:
indeed we could add a small section in troubleshooting doc about caching. It could include several potential issues: caching is not effective because cache folder is not mounted or has wrong permissions, one cpu it is not giving performance gain and hddl is not working.